server/models/Request.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema({
+  ownerId: {
```

Comment:
Consider renaming to `userId` if you referenced User in the schema.

Reply:
Hey Sunday! Since we have an owner and a sitter who are part of a booking request, I guess having this field as `ownerId` should be fine, thoughts? :)